sdk/highlight-react/src/components/ReportDialog/ReportDialog.tsx

```diff
@@ -76,119 +86,160 @@ const ReportDialog = ({
 	}
 
 	return (
-		<main className={`container`}>
-			<div className={`card`}>
-				<div
-					className={`cardContents ${
-						sentReport && 'cardContentsVisible'
-					}`}
-				>
-					<h1 className={`title`}>{successMessage}</h1>
-					<h4 className={`subtitle`}>{successSubtitle}</h4>
-					<button
-						className={`button confirmationButton`}
-						onClick={onCloseHandler}
-					>
-						Close
-					</button>
-				</div>
-				<div
-					className={`cardContents ${
-						!sentReport && 'cardContentsVisible'
-					}`}
-				>
-					<div>
-						<h1 className={`title`}>{title}</h1>
-						<h2 className={`subtitle`}>
-							{subtitle} {subtitle2}
-						</h2>
-					</div>
-					<form className={`form`} onSubmit={handleSubmit}>
-						<label>
-							{labelName}
-							<input
-								type="text"
-								value={name}
-								name="name"
-								autoFocus
-								onChange={(e) => {
-									setName(e.target.value)
-								}}
-							/>
-						</label>
-
-						<label>
-							{labelEmail}
-							<input
-								type="email"
-								value={email}
-								name="email"
-								onChange={(e) => {
-									setEmail(e.target.value)
-								}}
-							/>
-						</label>
-
-						<label>
-							{labelComments}
-							<textarea
-								value={verbatim}
-								placeholder={placeholderComments}
-								name="verbatim"
-								rows={3}
-								onChange={(e) => {
-									setVerbatim(e.target.value)
-								}}
-							></textarea>
-						</label>
-
-						<div className={`formFooter`}>
-							<div className={`formActionsContainer`}>
-								<button
-									type="submit"
-									className={
-										sendingReport
-											? `button loadingButton`
-											: 'button'
-									}
-								>
-									{labelSubmit}
-								</button>
+		<>
+			<style
+				dangerouslySetInnerHTML={{
+					__html: `
+					@font-face {
+						font-display: swap;
+						font-family: 'Steradian';
+						font-style: normal;
+						font-weight: normal;
+						src: local('Steradian Regular'), local('SteradianRegular'),
```

Comment:
chatted offline with @deltaepsilon about this - we want to inline this into the `.js` bundle so that a customer doesn't have to import a second `index.css` file. not a security concern here since this is a constant value.